plugin-kbindicator/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ find_package(PkgConfig REQUIRED)
 pkg_check_modules(XKB_COMMON REQUIRED xkbcommon)
 pkg_check_modules(XKB_COMMON_X11 QUIET xkbcommon-x11)
 
+include_directories(${XKB_COMMON_INCLUDEDIR})
```

Comment:
moving it inside `if(XKB_COMMON_X11_FOUND)` works for me too. however using `target_include_directories(${PLUGIN} ${XKB_COMMON_X11_INCLUDE_DIRS})` instead doesn't.